packages/appstore/src/modules/tour-guide/wallet-tour-guide.tsx

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
 import Joyride, { ACTIONS, EVENTS, STATUS } from 'react-joyride';
 import { getWalletStepConfig, getWalletStepLocale, wallet_tour_styles } from 'Constants/tour-steps-config';
+import { useAvailableWallets } from '@deriv/hooks';
 import { useStore, observer } from '@deriv/stores';
 
 const WalletTourGuide = observer(() => {
-    const { traders_hub } = useStore();
+    const { modules, traders_hub } = useStore();
     const { is_wallet_tour_open, toggleIsWalletTourOpen } = traders_hub;
+    const { current_list } = modules.cfd;
 
     const [joyride_index, setJoyrideIndex] = React.useState(0);
 
     const wallet_tour_step_locale = getWalletStepLocale();
 
+    const { data: sorted_wallets } = useAvailableWallets();
+
+    // if the user has an mt5 account, show a different message for the cfd step
+    const has_mt5_account = Object.keys(current_list)
+        .map(key => current_list[key])
+        .some(account => account.landing_company_short === 'svg' && account.market_type === 'synthetic');
```

Comment:
Maybe can check `useCFDAccounts` hook which returns cfd_real_accounts. This has all the deets in the cfd